src/main/scala/mesosphere/marathon/core/group/GroupManagerModule.scala

```diff
@@ -25,9 +24,10 @@ class GroupManagerModule(
     val groupManager = new GroupManagerImpl(config, None, groupRepo, scheduler)
 
     val startedAt = System.currentTimeMillis()
-    Kamon.metrics.gauge("service.mesosphere.marathon.uptime", Time.Milliseconds)(
-      System.currentTimeMillis() - startedAt
-    )
+    Metrics.gauge(
+      "marathon.uptime.duration",
```

Comment:
`marathon.uptime.seconds`

Reply:
If we follow the Prometheus naming guidelines, it should stay as is.

Reply:
In one place I appended `.seconds` in addition to `.duration`. But I am unsure if we need to do it. Maybe it is better to have a documented rule that durations are in seconds, and memory/space are in bytes by default everywhere (HTTP, StatsD, etc). What do you think?